anchor-reference-server/src/main/java/org/stellar/anchor/reference/AnchorEventServer.java

```diff
@@ -0,0 +1,102 @@
+package org.stellar.anchor.reference;
+
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.header.Header;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.springframework.beans.factory.DisposableBean;
+import org.springframework.kafka.support.serializer.JsonDeserializer;
+import org.springframework.stereotype.Component;
+import org.stellar.anchor.event.models.AnchorEvent;
+import org.stellar.anchor.event.models.QuoteEvent;
+
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Properties;
+
+@Component
+public class AnchorEventServer implements DisposableBean, Runnable {
+    private Thread thread;
+    private Consumer consumer;
+    private volatile boolean shutdown = false;
+
+    AnchorEventServer(){
+        this.thread = new Thread(this);
```

Comment:
The Java ExecutorService provides thread pool and easier to manage than the Thread. This may help the stability of the server.

https://www.baeldung.com/java-executor-service-tutorial

Reply:
will do this in a separate PR